core/src/main/scalajs/weaponregex/model/mutation/TokenMutatorJS.scala

```diff
@@ -24,11 +24,6 @@ case class TokenMutatorJS(tokenMutator: TokenMutator) {
   @JSExport
   val levels: js.Array[Int] = tokenMutator.levels.toJSArray
 
-  /** A short description of the mutator
-    */
-  @JSExport
-  val description: String = tokenMutator.description
```

Comment:
Technically a breaking change, but I checked, and it's not used in stryker-js, so I think we can get away with it.

Could you also remove it from the `TokenMutator` interface in the `index.d.ts` file?

Reply:
Perfect 👌 I'll update the PR later.